README.md

```diff
@@ -86,6 +86,44 @@ This code snippet will output the translated strings such as the following:
     }
 ```
 
+### Translation Requests
```

Comment:
you can probably most easily view the final README by hitting the View button above ☝️